src/sound/soundbase.h

```diff
@@ -168,9 +168,9 @@ class CSoundBase : public QThread
 
 signals:
     void ReinitRequest ( int iSndCrdResetType );
-    void ControllerInFaderLevel ( int iChannelIdx, int iValue );
-    void ControllerInPanValue ( int iChannelIdx, int iValue );
-    void ControllerInFaderIsSolo ( int iChannelIdx, bool bIsSolo );
-    void ControllerInFaderIsMute ( int iChannelIdx, bool bIsMute );
+    void ControllerInFaderLevel ( int iMIDIIdx, int iValue );
```

Comment:
`CMidiCtlEntry` should have the first entry renamed to match if it's not `iChannel` any more.  Or else rename it to `iChannel` -- it's the value of `CMidiCtlEntry::iChannel`, after all...

Reply:
It's really a "virtual channel", where 0 is _our_ fader, and the others are allocated in server order.

In MIDI-speak it's not a channel anyway, it's a controller offset. We are just using it to access a Jamulus channel.

I'm happy to rename items if you have suggestions, but I still think the architecture is ok.

Reply:
Well, whatever. My problem with the `z` solution is the duplicate fader. That's a show-stopper for me, and why I put in a lot of work to come up with an improved solution.

I'm going to leave this for a while.